StreamChatClient_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -83,6 +82,19 @@ extension NSManagedObjectContext {
         return dto
     }
     
+    func saveChannel<ExtraData: ExtraDataTypes>(payload: ChannelPayload<ExtraData>,
+                                                query: ChannelListQuery?) throws -> ChannelDTO {
+        let dto = try saveChannel(payload: payload.channel, query: query)
+        
+        // Sometimes, `members` are not part of `ChannelDetailPayload` so they need to be saved here too.
```

Comment:
This is weird we have to double check where the source of truth. Backend should fix that or explain how it works.

Reply:
For example, the channel query payload returns `members` next to the channel payload, but the even JSON payload have them inside.